templates/vue/src/components/TapestryMain/index.vue

```diff
@@ -13,7 +13,7 @@
           :target="nodes[link.target]"
         ></tapestry-link>
       </g>
-      <g v-if="dragSelectEnabled && dragSelectReady" class="nodes">
+      <g v-if="!dragSelectEnabled || dragSelectReady" class="nodes">
```

Comment:
By negating the `dragSelectEnabled` check, I think you are also unintentionally causing the nodes to show up if `this.settings.renderMap` is true. i.e. `!!this.settings.renderMap === this.settings.renderMap`

I think this issue would be solved if you just add `!this.settings.renderMap` to this `v-if`.

```suggestion
      <g v-if="!this.settings.renderMap && (!dragSelectEnabled || dragSelectReady)" class="nodes">
```

Reply:
Actually, I found out that we are rendering TapestryMap instead of the TapestryMain component when `this.settings.renderMap` is true, so `this.settings.renderMap` is guaranteed to be false in TapestryMain. I will remove the check for `this.settings.renderMap` from TapestryMain instead, since it is not needed.